src/main/features/settlement-agreement/views/settlement-agreement-confirmation.njk

```diff
@@ -0,0 +1,37 @@
+{% extends "layout.njk" %}
+{% from "support.njk" import supportWidget %}
+{% from "externalLink.njk" import externalLink %}
+
+{% extends "layout.njk" %}
+
+{% block content %}
+<div class="grid-row">
+  <div class="column-two-thirds">
+
+    <div class="govuk-box-highlight">
+
+      {% if claim.settlement.isOfferRejected() %}
+        <h1 class="bold-large">{{ t('You’ve rejected the settlement agreement') }}</h1>
+      {% endif %}
+
+    </div>
+
+    <h2 class="heading-medium">{{ t('What happens next') }}</h2>
+
+    {% if claim.settlement.isOfferRejected() %}
+      <p>{{ t('The claimant can request a County Court Judgment (CCJ) against you. A CCJ would order you to repay the money in line with the terms of the agreement.') }}</p>
+      <p>{{ t('If they request a CCJ, you can ask a judge to consider changing the repayment plan, based on your financial details.') }}</p>
+      <p>{{ t('The court has reviewed the plan and believes you can afford it, so a judge may not change it.') }}</p>
+      <p>{{ t('We’ll email you when the claimant responds.') }}</p>
+    {% endif %}
+
+
+    {% include "includes/service_survey_link.njk" %}
+
+    <p><a class="button" href="{{ DashboardPaths.dashboardPage.uri }}">{{ t('Go to your account') }}</a></p>
```

Comment:
switch to internal link and remove the p. if it needs padding add a div with a form-group